teps/0048-task-results-without-results.md

```diff
@@ -4,9 +4,10 @@ authors:
   - "@pritidesai"
   - "@jerop"
   - "@vinamra28"
+  - "@bobcatfish"
 creation-date: 2020-10-20
-last-updated: 2022-08-09
-status: implementable
+last-updated: 2023-02-13
+status: proposed
```

Comment:
why is this proposal going from implementable to proposed?

Reply:
earlier we proposed on another solution - `Specifying default values for Results at the time of authoring the task` but later on @jerop and @bobcatfish had some discussion around this and we came up with new proposal of specify default results at the time of variable substitution

Reply:
why not leave as implementable though? This seems pretty fleshed out, what additional info would be needed to mark it as implementable?

Reply:
agreed, let's leave it as implementable